tools/UnstandardnessTestForDM/UnstandardnessTestForDM/Form1.cs

```diff
@@ -1,484 +0,0 @@
-﻿using System;

```

Comment:
Can we keep the line checker at least? This is nice for running every once in awhile and seeing how many lines of code we have.